Packs/GLPI/Integrations/GLPI/GLPI.py

```diff
@@ -0,0 +1,973 @@
+        entries = []
+        demisto.debug('fetch files')
+        ticket_docs = client.get_ticket_docs(ticket_id)
+        if ticket_docs:
+            for ticket_doc in ticket_docs:
+                if ticket_doc.get('date_mod') > formated_date:
+                    document = client.get_item('Document', ticket_doc.get('documents_id'))
+                    if '_mirrored_from_xsoar' not in document.get('filename'):
+                        file = client.download_document(ticket_doc.get('documents_id'), filename=document.get('filename'))
+                        demisto.debug('file fetched')
+                        filepath, filename = os.path.split(file)
+                        f = open(file, 'rb')
+                        data = f.read()
+                        entries.append(fileResult(filename, data))
+
+        comments_result = client.get_ticket_comments(ticket_id)
+        if comments_result:
+            for note in comments_result:
+                if 'Mirrored from Cortex XSOAR' not in note.get('content') and note.get('date_mod') > formated_date:
+                    comments_context = {'comments_and_work_notes': unescape(note.get('content'))}
+                    entries.append({
+                        'ContentsFormat': formats['html'],
+                        'Type': entryTypes['note'],
+                        'Contents': unescape(note.get('content')),
+                        'Note': True,
+                        'EntryContext': comments_context
+                    })
+        demisto.debug(f'Pull result is {new_incident_data}')
+        return GetRemoteDataResponse(new_incident_data, entries)
+    except Exception as e:
+        raise DemistoException('Error : ' + str(e))
+
+
+def update_remote_system_command(client: Client, args: Dict[str, Any], params: Dict[str, Any]) -> str:
+    """update-remote-system command: pushes local changes to the remote system
+
+    :type client: ``Client``
+    :param client: XSOAR client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['data']`` the data to send to the remote system
+        ``args['entries']`` the entries to send to the remote system
+        ``args['incidentChanged']`` boolean telling us if the local incident indeed changed or not
+        ``args['remoteId']`` the remote incident id
+
+    :return:
+        ``str`` containing the remote incident id - really important if the incident is newly created remotely
+
+    :rtype: ``str``
+    """
+    parsed_args = UpdateRemoteSystemArgs(args)
+    ticket_id = parsed_args.remote_incident_id
+    if parsed_args.delta:
+        demisto.debug(f'Got the following delta keys {str(list(parsed_args.delta.keys()))}')
+
+    demisto.debug(f'Sending incident with remote ID [{parsed_args.remote_incident_id}] to remote system\n')
+    new_incident_id: str = parsed_args.remote_incident_id
```

Comment:
What is the difference between `ticket_id` and `new_incident_id`?

Reply:
remove ticket_id